.ui_scaffold/common/src/main.tsx

```diff
@@ -0,0 +1,11 @@
+// @ts-nocheck
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App'
+import "@fuel-ui/css";
```

Comment:
What's the purpose of the `css`? Afaik it's for creating custom css inside the code but I don't expect us to be doing that. It doesn't hurt to have it I guess just in case someone wants to use it.

Reply:
Many components that will most likely be used have `css` controls (https://fuellabs.github.io/fuel-ui/) for seemingly common operations, I used at least one `css` field for even simple UIs I've been building.